css/style.css

```diff
@@ -1,4 +1,23 @@
-#card-container{
+body {
```

Comment:
Should we use body here? Would it override the existing style on the page?

Reply:
Oh good point!! For some reason i thought that our styles wouldn't override squarespace but I just realized we specifically write a style at the bottom to override squarespace. Updated :)